### DISCUSSION TITLE
Batch events in single mail.

### DISCUSSION THREAD

**Discussion by Odraio** [datalust/seq-tickets#2212] [Help]
The documentation part ['Notifying with Email'](https://docs.datalust.co/docs/formatting-html-email) refers to [Seq.App.Mail.Smtp](https://www.nuget.org/packages/seq.app.mail.smtp). I would like to batch events in a single email, but the described package doesn't have such option. The [Seq.App.DigestEmail](https://github.com/datalust/seq-app-digestemail) package provides the functionality, but hasn't been updated for years. 

Are there any plans to provide an updated package which support batching of events in a single email?

**Comment by nblumhardt** [comment]
Hi Raymond, thanks for your message.

Alert notifications (rather than directly streamed events) are the intended mechanism for batching multiple items into a single email. The "include contributing events" option provides a summary of the events behind an alert, and the aggregation-based alerting mechanism itself is designed for handling notifications over arbitrarily large numbers of events.

Ideally, we'd get the alerting functionality lined up with what you're aiming to achieve; can you possibly describe your use case in a bit more detail?

Thanks again,
Nick
